TreeTracker/UI/Views/Notes/NotesViewController.swift

```diff
@@ -64,7 +64,7 @@ private extension NotesViewController {
         let doneButton: UIBarButtonItem = UIBarButtonItem(
             title: L10n.Keyboard.Toolbar.DoneButton.title,
             style: .done,
-            target: view,
+            target: self,
```

Comment:
Appreciate you may be fixing a bug here, but we should try keep our changes small where possible. Things like this could be a separate change request.

Reply:
I think that's your code from the main branch, maybe it's showing as a change because the "Greenstand:feature/messaging" branch is 4 commits behind main. 
If it's better I can change the pull request to the main branch

Reply:
Ah ok thanks - i'll update the feature branch